_includes/_components/doctrine-events.html

```diff
@@ -45,4 +79,4 @@
             </div>
```

Comment:
can you add a link to the github repo like this : 

https://github.com/KnpLabs/rad-components-site/pull/4/files#diff-61a72530a68f2cb152bb0907aa033b19R71